Migrations/20200507151313_TestTypeToTestResult.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace CoviIDApiCore.Migrations
+{
+    public partial class TestTypeToTestResult : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<int>(
+                name: "TestType",
```

Comment:
Should that not be an enum reading as string, instead of an int?

Reply:
Ahh you're right, forgot the conversion, will add it in